evap/student/templates/student_vote.html

```diff
@@ -312,34 +312,73 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
             document.querySelectorAll("[data-mark-no-answers-for]").forEach(button => {
                 const contributorId = button.dataset.markNoAnswersFor;
                 const voteArea = document.getElementById(`vote-area-${contributorId}`);
+                const markNoAnswerModal = voteArea.querySelector(".mark-no-answer-modal");
+                const textareaAnswerFields = voteArea.querySelectorAll(".vote-inputs textarea");
                 const collapseToggle = voteArea.closest(".collapsible").querySelector(".collapse-toggle");
 
-                button.addEventListener("click", () => {
+                const onButtonClicked = () => {
```

Comment:
This name is not really accurate anymore, how about
```suggestion
                const clearContributorAnswers = () => {
```